LEAF_Request_Portal/templates/print_form.tpl

```diff
@@ -676,20 +731,20 @@ function admin_changeInitiator() {
     dialog.indicateBusy();
 
     dialog.setSaveHandler(function() {
-    	if($('#changeInitiator').val() != '') {
+        if($('#changeInitiator').val() != '') {
             $.ajax({
                 type: 'POST',
                 url: './api/form/<!--{$recordID|strip_tags}-->/initiator',
                 data: {CSRFToken: CSRFToken,
-                	   initiator: $('#changeInitiator').val()},
+                       initiator: $('#changeInitiator').val()},
                 success: function() {
```

Comment:
fail catch.  I know it doesn't look like this one _needs_ one.  But for troubleshooting purposes.